web/js/palettes/legend.js

```diff
@@ -136,14 +135,15 @@ export function palettesLegend(spec) {
       $colorBox.on('mouseenter', showClassUnitHover);
       $colorBox.on('mouseout', hideUnitsOnMouseOut);
     });
+    // TODO: Review this each loop. It can probably be removed.
     var $detailPanel = $('<div></div>');
     loEach(legend.colors, function (color, classIndex) {
       var label = legend.tooltips[classIndex];
       label = (legend.units) ? label + ' ' + legend.units : label;
       var $row = $('<div></div>')
-        .addClass('wv-palettes-class-detail')
+        .addClass('$colorBox')
```

Comment:
Why are you changing the class on each row to `$colorBox` instead of `wv-palettes-class-detail`?

Reply:
This was accidentally introduced. Good catch.